src/nl/documentatie/10.woocommerce-v5.0.md

```diff
@@ -0,0 +1,329 @@
+
+::: note
+Let op! Zorg dat "Direct verwerken" is ingeschakeld, wanneer je deze optie gebruikt in combinatie met de Track & Trace in email optie. Anders wordt de Track & Trace code niet opgenomen in de email naar je klant, omdat er op het moment van verzenden nog geen barcode is gecreëerd.
+:::
+
+##### Track & Trace in account
+
+Track & Trace in account inschakelen. Als je klanten een account kunnen aanmaken in je webshop, kun je met er met deze optie voor zorgen dat de link van de barcode van de bestelling in het account wordt getoond.
+
+##### Barcode in notitie
+
+Barcode in notitie inschakelen
+
+##### Barcode in notitie titel
+
+Je kunt hier de tekst veranderen dat voor de barcode van de zending komt te staan in de notities van de bestelling. Dit is de tekst die voor de barcode komt te staan in de notities.
+
+::: note
+Let op: maak je gebruik van een koppeling met Effect Connect om Bol.com-orders in jouw WooCommerce-omgeving te krijgen? Laat de standaard waarde Tracking code: staan om de barcode automatisch terug te koppelen naar Bol.com.
+:::
+
+<hr/>
+<br/>
+
+### Bestellingen
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_b_nl.jpeg" alt="woocommerce plugin bestellingen" />
+
+#### Automatisering orderstatus
+
+##### Klantadres opslaan in adresboek
+
+Klantadressen van bestellingen opslaan in je MyParcel adresboek
+
+##### Bestelstatus bij label aanmaken
+
+Selecteer de order status die gebruikt moet worden wanneer een label wordt aangemaakt
+
+##### Bestelstatus bij label scannen
+
+Selecteer de order status die gebruikt moet worden wanneer een label is gescand
+
+##### Bestelstatus bij bezorging
+
+Selecteer de order status die gebruikt moet worden wanneer een zending is afgeleverd.
+
+##### Bestelstatus mail
+
+Selecteer de order status die gebruikt moet worden voor de Track & Trace e-mail.
+
+###### Stuur notificatie na
+
+Selecteer het aantal dagen na welke de Track & Trace e-mail verstuurd moet worden
+
+##### Digitale postzegel bestelstatus
+
+Selecteer de order status die gebruikt moet worden voor digitale postzegels.
+
+### Standaard gewichten
+
+#### Basisgewicht pakket(g)
+
+Geef hier aan wat het gewicht van een leeg pakket is. Het ligt aan de instellingen van WooCommerce of dit in grammen of kilo's is.
+
+#### Basisgewicht brievenbuspakje (g)
+
+Geef hier aan wat het gewicht van een leeg brievenbuspakket is. Het ligt aan de instellingen van WooCommerce of dit in grammen of kilo's is.
+
+#### Basisgewicht digitale postzegel (g)
+
+????
+
+<hr/>
+<br/>
+
+### Labels
+
+<MPImg src="/documentation/woocommerce/nl/myparcel_wc_5_c_nl.jpeg" alt="woocommerce plugin labels" />
+
+#### Omschrijving
+
+Omschrijving op het label
```

Comment:
Veel 'uitleg' is nogal summier. Dat is wellicht een bewuste keuze. Maar bij deze zou ik als plugin gebruiker toch wel echt graag wat meer zien. Bijvoorbeeld:
1) max lengte ongeveer 45 tekens (afh van carrier, wordt automatisch afgekapt)
2) mogelijke 'variabelen': [DELIVERY_DATE]: Bezorgdatum, [ORDER_NR]: Ordernummer, [PRODUCT_ID]: Product ID, [PRODUCT_NAME]: Productnaam, [PRODUCT_QTY]: Productaantal, [PRODUCT_SKU]: Product SKU, [CUSTOMER_NOTE]: Opmerking klant